plane/source/docs/guide-tailsitter.rst

```diff
@@ -50,7 +53,52 @@ QHOVER and QLOITER.
 .. youtube:: bMsfjwUAfkM
     :width: 100%
 
-However, it can also have copter-like motors, like a conventional QuadPlane if :ref:`Q_TAILSIT_MOTMX<Q_TAILSIT_MOTMX>` is non-zero. Then this parameter determines which motors remain active in normal forward flight (plane modes). If non-zero, then use the :ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>` parameter to configure the multicopter motor style, and the appropriate MOTORx outputs will be activated.
+
+However, it can also have copter-like motors, like a conventional QuadPlane if :ref:`Q_TAILSIT_MOTMX<Q_TAILSIT_MOTMX>` is non-zero. Then this parameter determines which motors remain active in normal forward flight (plane modes). If non-zero, then use the :ref:`Q_FRAME_CLASS<Q_FRAME_CLASS>` and :ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>` parameter to configure the multicopter motor style, and the appropriate MOTORx outputs will be activated.
+
+.. note:: Currently, only the following combinations are supported for CopterMotor tailsitters:
+
++------------------------+------------------------------------+----------------------------------+
+| Motors                 |:ref:`Q_FRAME_CLASS<Q_FRAME_CLASS>` |:ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>` |
++------------------------+------------------------------------+----------------------------------+
+|  Quad                  | 1                                  | 0, 1                             |
++------------------------+------------------------------------+----------------------------------+
+|  Tri                   | 7                                  | ignored                          |
++------------------------+------------------------------------+----------------------------------+
+
+Motor Rotation and Layout
+=========================
+
+(looking down on nose from above)
+
+Single/Dual Motor Tailsitter
+----------------------------
+
+Motors are controlled by the Throttle, Throttle Left, Throttle Right outputs:
+
+.. image:: ../images/tailsit-motors.jpg
+
+
+QUAD I Motor Tailsitter
+-----------------------
+Motors are controlled by the M1-M4 outputs:
+
+.. image:: ../images/i-copter-quadplane.jpg
+
+
+QUAD X Motor Tailsitter
+-----------------------
+Motors are controlled by the M1-M4 outputs:
+
+.. image:: ../images/x-copter-quadplane.jpg
+
+
+Tricopter Tailsitter
+--------------------
+Motors are controller by the M1/M2/M4 outputs:
```

Comment:
the only testing I know of for this configuration is my SITL testing, which has motors 1 and 2 on the wings and motor 4 on the top of the tail fin

Reply:
yes, need to swap 1 and 2...

Reply:
did that